README.md

```diff
@@ -72,6 +85,11 @@ Note: If you run into problems, it's possible the version of Mercurial you are u
 
 Note: the script will take a long time to run (on my laptop more than 7 hours). If you want to test a simple change and you don't intend to actually mine the data, you can modify the repository.py script to limit the number of analyzed commits. Simply add `limit=1024` to the call to the `log` command.
 
+### Training the defect model
+
+To train the model for mode `defect`:
+
+    python3 -m scripts.trainer defect
```

Comment:
Could you move this to the `Usage` section? And maybe rename `Usage` to something like `Usage (training a model)`.

Reply:
done